libherokubuildpack/src/buildpack_output/mod.rs

```diff
@@ -560,6 +651,38 @@ mod test {
     use libcnb_test::assert_contains;
     use std::fs::File;
 
+    #[test]
+    fn background_timer() {
+        let io = BuildpackOutput::new(Vec::new())
+            .start_silent()
+            .section("Background")
+            .start_timer("Installing")
+            .finish()
+            .finish()
+            .finish();
+
+        // Test human readable timer output
+        let expected = formatdoc! {"
+            - Background
+              - Installing ... (< 0.1s)
+            - Done (finished in < 0.1s)
+        "};
+
+        assert_eq!(
+            expected,
+            strip_ansi_escape_sequences(String::from_utf8_lossy(&io))
+        );
+
+        // Test timer dot colorization
+        let expected = formatdoc! {"
+            - Background
+              - Installing\u{1b}[2;1m .\u{1b}[0m\u{1b}[2;1m.\u{1b}[0m\u{1b}[2;1m. \u{1b}[0m(< 0.1s)
```

Comment:
I agree that since the logic lives here (to specify what to dim) it should be tested here. I still want to make sure the output looks okay as human readable, so I'm testing once with and once without color codes.